packages/accordion/src/js/jquery.js

```diff
@@ -0,0 +1,86 @@
+/*! [replace-name] v[replace-version] */
+/***************************************************************************************************************************************************************
+ *
+ * Accordion function
+ *
+ * A component to allow users to show or hide page elements.
+ *
+ **************************************************************************************************************************************************************/
+
+
+/**
+ * The accordion jquery function to add attributes and event listeners
+ *
+ * Public functions are:
+ * $('.js-uikit-accordion-wrapper').uikitAccordion().isOpen()
+ * $('.js-uikit-accordion-wrapper').uikitAccordion().setA11y()
+ * $('.js-uikit-accordion-wrapper').uikitAccordion().addListeners()
+ */
+$.fn.uikitAccordion = function() {
+	var $elements = this;
+
+	return {
+		/**
+		 * Check an accordion for its state
+		 *
+		 * @return {Boolean} - true = open; false = closed
+		 */
+		isOpen: function() {
+			return !$elements.children('.js-uikit-accordion').hasClass('uikit-accordion--closed');
```

Comment:
We actually have to as the html can come with either a open or closed default. It is determined by the class so that it all still works when js is off :)